binaryaudit/db.py

```diff
@@ -169,7 +171,9 @@ def insert_ba_transaction_details(self,
         inserts new object to the [details table]
         '''
         session = self._acquire_session()
+        date = datetime.utcnow()
```

Comment:
Shouldn't recording date once per transaction be enough? As the analysis is always done based on the build base, all the artifacts would be there at the same time. Seems another workflow wouldn't be really in use?

Thanks

Reply:
Thinking more, probably there could be interest if artifacts get longer to produce, etc. This will cover that scenario, so please nevermind.

Thannks.